snmp/datadog_checks/snmp/data/default_profiles/_generic-entity-sensor-mib.yaml

```
@@ -0,0 +1,97 @@
+metrics:
+  - MIB: ENTITY-SENSOR-MIB
+    table:
+      OID: 1.3.6.1.2.1.99.1.1
+      name: entPhySensorTable
+    symbols:
+      - OID: 1.3.6.1.2.1.99.1.1.1.4
+        name: entPhySensorValue
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.2.1.99.1.1.1.1
+          name: entPhySensorType
+        tag: sensor_type
+        mapping:
+          1: other
+          2: unknown
+          3: volts_ac
+          4: volts_dc
+          5: amperes
+          6: watts
+          7: hertz
+          8: celsius
+          9: percent_rh
+          10: rpm
+          11: cmm
+          12: truthvalue
+      - column:
+          OID: 1.3.6.1.2.1.99.1.1.1.2
+          name: entPhySensorScale
+        tag: sensor_scale
+        mapping:
+          1: yocto
+          2: zepto
+          3: atto
+          4: femto
+          5: pico
+          6: nano
+          7: micro
+          8: milli
+          9: units
+          10: kilo
+          11: mega
+          12: giga
+          13: tera
+          14: exa
+          15: peta
+          16: zetta
+          17: yotta
+      - column:
+          OID: 1.3.6.1.2.1.99.1.1.1.3
+          name: entPhySensorPrecision
+        tag: sensor_precision
+      - column:
+          OID: 1.3.6.1.2.1.99.1.1.1.6
+          name: entPhySensorUnitsDisplay
+        tag: sensor_units
+      - column:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.2
+          name: entPhysicalDescr
+        tag: entity_description
+      - column:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.5
+          name: entPhysicalClass
+        tag: entity_class
+        mapping:
+          1: other
+          2: unknown
+          3: chassis
+          4: backplane
+          5: container
+          6: power_supply
+          7: fan
+          8: sensor
+          9: module
+          10: port
+          11: stack
+          12: cpu
+      - column:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.7
+          name: entPhysicalName
+        tag: entity_name
```

```suggestion
        tag: entity_physical_name
```

I'm wondering if we should always derive the tag from the name of the symbol for consistency? WDYT?

---

No strong opinion on this particular case. Adding `physical` is longer but clearer so good for me 👍